software/o_c_REV/HEM_VectorLFO.ino

```diff
@@ -158,10 +182,13 @@ private:
         gfxInvert(1, 14, 7, 9);
 
         gfxPrint(10, 15, ones(freq[ch]));
-        gfxPrint(".");
-        int h = hundredths(freq[ch]);
-        if (h < 10) gfxPrint("0");
-        gfxPrint(h);
+        if (freq[ch] < max_freq) {
```

Comment:
Why have this `if` when `freq` is constrained?

Reply:
It was an edge case for when freq[ch] == max_freq. The extra 0 would cause the text to spill over. I changed it now to just skip displaying decimal places when freq >= 10Hz.